src/compiler/binder.ts

```diff
@@ -3637,6 +3637,10 @@ namespace ts {
         let excludeFlags = TransformFlags.NodeExcludes;

```

Comment:
@joeywatts I think we can merge this soon. Would help if you could:
- add a little bit more to the PR description: 
    - what were some judgment calls or design decisions made in this PR?
- rebase this down to a few smaller commits

Reply:
✔️